### DISCUSSION TITLE
How to connect Mizu wallet with existing Petra wallet address on Telegram Mini app

### DISCUSSION THREAD

**Discussion by CryptainGolD** [aptos-labs/aptos-developer-discussions#466] [Dev Doc and Tutorial Feedback]
This template works to generate a aptos wallet address  and Only send and swap coins. Can I make it connect with existing aptos address from my Petra wallet and works all of Petra wallet(Send, swap, View collections etc)?

**Comment by WGB5445** [comment]
This won't work; they don't use the same technology at the foundational level, such as private keys or mnemonic phrases, so it's completely unfeasible.

**Reply by CryptainGolD** [reply]
Thanks for your help
